solutions/cf-1398C.mdx

```diff
@@ -99,4 +99,28 @@ public class GoodSubarrays {
 ```
 </JavaSection>
 
+<PySection>
+```py
+# pretty much the official editorial sol
+n = int(input())
+for i in range(n):
```

Comment:
Change these to `t` perhaps? Then instead of `len_nums` use `n`.

Reply:
agree with the first, disagree with the second